### ISSUE TITLE
Expand Details not showing up as possible action in voice over

### ISSUE THREAD

**Issue by ClementineHahn** [cds-snc/design-gc-conception#482]
**📆 Study info**
- Accessibility testing - July 2023 - Facilitator: @Lu-stack-art & @ClementineHahn 
- Name of the test : 
- Triage Board : _link to the associated trello board_

**🚦 Issue severity** :  Severe  

**📝 What happened?**
User is prompted to find the recipe section of the homepage, and using the alt nav/verbal commands/numerical codes, to locate and click on tabs. The mouse does not respond to users voice. Generally the default command is "scroll down" and system was not responsive to the users voice/or this command.

**📗 User story**
_As a user, I need  to ....., so that...._

**📜 Recommendations**
Recommendation was to use other words, to get a response from the tech. "move down" was more responsive, but it was a slight improvement (mouse would make small movements up and down the screen) while user had to repeat the "move down" command multiple times, until they got to where they needed to be on the page. (taxing and tedious experience for users).

**ℹ️ More info**
https://trello.com/c/fShhZfEK

**Comment by Lu-stack-art** [comment]
Fake App Scenario- User Moderated testing for Alternative Navigation

https://trello.com/c/fShhZfEK

Issue to be reviewed and resolved is moderate to severe

User is prompted to find the recipe section of the homepage, and using the alt nav/verbal commands/numerical codes, to locate and click on tabs. The mouse does not respond to users voice. Generally the default command is "scroll down" and system was not responsive to the users voice/or this command.

Recommendation was to use other words, to get a response from the tech. "move down" was more responsive, but it was a slight improvement (mouse would make small movements up and down the screen) while user had to repeat the "move down" command multiple times, until they got to where they needed to be on the page. (taxing and tedious experience for users).

**Comment by Lu-stack-art** [comment]
Fake App Scenario-User Moderated testing for Screen Reader

https://trello.com/c/mK2JMSj9

User is prompted to locate the allergy considerations on the page. The expectation was for user to locate this section and then detailed content would display itself, beneath the heading/navigation link/tab. User was unable to easily locate the allergy section, as the tech didn't detect it, and when researcher explains that it is a link, the user retraces their steps and double taps on the link, and details do not drop down. The voice over tech is unable to read or clearly detect this section. 

Researcher give hints which directed user to the navigation link/tab, prompting user to double tap. Otherwise this section would easily be missed, using this assertive tech. The drop down is not detectable or at least not clearly by the voice over. The expectation would be the (open/close). And when user located the section, the voice over did not indicate that the screen change or new content was added.
